### ISSUE TITLE
"make deploy" fails

### ISSUE THREAD

**Issue by pires** [mesosphere/dcos-kubernetes-quickstart#91]
The following command:

```
KUBERNETES_FRAMEWORK_VERSION=1.0.3-1.9.7 make deploy
```

resulted in the following failure:

```
Waiting for DC/OS Master to be ready...

/Users/pires/Work/mesosphere/dcos-kubernetes-quickstart/./dcos package install --yes marathon-lb;\
	/Users/pires/Work/mesosphere/dcos-kubernetes-quickstart/./dcos package install --yes kubernetes --package-version=1.0.3-1.9.7 --options=./.deploy/options.json;\
	/Users/pires/Work/mesosphere/dcos-kubernetes-quickstart/./dcos marathon app add ./.deploy/kubeapi-proxy.json
By Deploying, you agree to the Terms and Conditions https://mesosphere.com/catalog-terms-conditions/#community-services
We recommend at least 2 CPUs and 1GiB of RAM for each Marathon-LB instance.

*NOTE*: For additional ```Enterprise Edition``` DC/OS instructions, see https://docs.mesosphere.com/administration/id-and-access-mgt/service-auth/mlb-auth/
Installing Marathon app for package [marathon-lb] version [1.12.2]
Marathon-lb DC/OS Service has been successfully installed!
See https://github.com/mesosphere/marathon-lb for documentation.
By Deploying, you agree to the Terms and Conditions https://mesosphere.com/catalog-terms-conditions/#certified-services
Kubernetes on DC/OS.

	Documentation: https://docs.mesosphere.com/service-docs/kubernetes
	Issues: https://github.com/mesosphere/dcos-kubernetes-quickstart/issues
Installing Marathon app for package [kubernetes] version [1.0.3-1.9.7]
Installing CLI subcommand for package [kubernetes] version [1.0.3-1.9.7]
New command available: dcos kubernetes
DC/OS Kubernetes is being installed!
Can't read from resource: ./.deploy/kubeapi-proxy.json.
Please check that it exists.
make: *** [install] Error 1
```

**Comment by rimusz** [comment]
latest quickstart does not support versions below 1.1.1-1.10.4

**Comment by rimusz** [comment]
is there the reason you running it with 1.0.3-1.9.7?

**Comment by pires** [comment]
Yes, was trying to reproduce a report where the user couldn't update from 1.0.3-1.9.7 to 1.1.1-1.10.4.

Can you please document in the README that the quickstart doesn't support any version before 1.1.1-1.10.4?

**Comment by pires** [comment]
Also, can you please open a PR that updates `/docs/cncf_conformance.md`, accordingly? Please, make sure you run the instructions before opening the PR.

**Comment by rimusz** [comment]
shall I make one PR for the readme and conformance docs update?

**Comment by pires** [comment]
Whatever you prefer. I like that idea.

**Comment by rimusz** [comment]
addressed with #92
